apps/minaintyg/src/pages/Certificates/components/CertificateListFilter/CertificateStatusFilter.tsx

```diff
@@ -0,0 +1,18 @@
+import { Select } from '@frontend/components'
+import { CertificateStatus } from '../../../../schema/certificateList.schema'
+import { useAppDispatch, useAppSelector } from '../../../../store/hooks'
+import { update } from '../../../../store/slice/certificateFilter.slice'
+
+export function CertificateStatusFilter({ options }: { options: CertificateStatus[] }) {
+  const { statuses } = useAppSelector((state) => state.certificateFilter)
```

Comment:
Is it better to send the values for each filter as prop from parent and have the appSelector only in the parent component?

Reply:
The effect would be the same, only we would need to add a property for every filter to pass the value down to the parent and move the tests to the parent for every filter, this would add a bit more code.

One thing I noticed now however is that we should pass the value and not the object from `useSelector` to avoid unnecessary re-renders since `useSelector` evaluates changes with a strict `===` operator.

I'll change them to be:
```typescript
const statuses = useAppSelector((state) => state.certificateFilter.statuses)
```